software/tobuild.txt

```diff
@@ -10,6 +10,6 @@ drivers
 utils
 perception
 motion_estimate
-control
 atlas-collection
+control
```

Comment:
Required for:

    - AtlasStateExtraCoder (used by calibrateAtlasArmEncoders.m to get arm joint encoder values)
    - AtlasFallDetector uses atlas lcm types
    - AtlasBehaviorModeListener uses atlas lcm types